platform/ios/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [CONTRIBUTING.md](../../CONTRIBUTING.md) to get started.
 
+## master
+
+### Other changes
+
+* `[MGLLoggingConfiguration sharedConfiguration].loggingLevel` now can filter logs from core. [#15120](https://github.com/mapbox/mapbox-gl-native/pull/15120)
```

Comment:
We've also changed the logging order (to better match core) - we should note this.